### ISSUE TITLE
As a User, I want to edit and delete previously recorded symptoms so that I can correct any errors or add anything I forgot to add.

### ISSUE THREAD

**Issue by BetsyRowley** [BetsyRowley/menstrual-tracker#9]
### Feature Description

- Input: User edits weight, rates mood (happy vs. sad), bloating, cramping, headaches, irritability, energy level, anxiety, breast tenderness, or appetite, or adds/edits comments regarding observations for the selected day.
- Activation: Clicking on the 'Edit Daily Observation' button.
- Action: An existing daily record is modified.
- Output: A message to the user that a 'Daily Observation' has been Edited and the record is modified in the database.

### User Case

| User Action | Action Result |
| ------------- | ------------- |
| Click 'Edit Daily Observation'  | Modal form opens on screen  |
| Edit Observation values  | Values Reflected in Modal form |
| Click 'Submit Observations'  | Daily Observation modified in DB & Modal confirmation shows on screen  |
| Closes Confirmation Modal | Modal Confirmation closes, Dashboard displayed  |

### Noun Extraction
* User
* Heath Symptoms --> DailyRecord

| Class Name: DailyRecord |   |
| ------------- | ------------- |
| Responsibilities  | Collaborators |
| **Knows (Variables)**: date, health values, and comments. **Can (Methods)**: can get & set Date, Health Values, and comments | User |

### Narration

**User**: Actually, Viewer. I need to update Daily Record for today, I forgot to add a comment.
**Viewer**: Controller, can you help me modify the Daily Record for today?
**Controller**: Model, request for you to modify today's Daily Record.
**Model**: My pleasure. Today's Daily Record retrieved and edited.
**Controller**: Thank you! Viewer, please relay that the update has been made.
**Viewer**: User, thanks for the update. Today's record has been corrected. Now, go enjoy your day!
**User**: Thank ye kindly, Viewer. I appreciate the speedy and reliable assistance.

**Comment by cklukas16** [comment]
@BetsyRowley Ok. I'm pretty content with where this story is at. 
Accomplished:
 - can edit from the list or daily view
 - previous entry data does populate the form
 - can delete the entry
 - ADDING ENTRIES: I added some client side validation so that future date entries are not allowed. I'm 99.5% sure this works as intended. 
 - DAILY INFO: I added some row/column formatting. 

Not Accomplished:
 - I was not able to pre-set the slider bars. I don't expect to solve this problem by our due date. 
 - pre-populated values are the ENUM values rather than integers. I'm ok with this in our final submission if you are. 

You ok if I merge with the main branch?
